api/envoy/config/bootstrap/v3/bootstrap.proto

```diff
@@ -301,7 +301,8 @@ message Admin {
   option (udpa.annotations.versioning).previous_message_type = "envoy.config.bootstrap.v2.Admin";
 
   // Configuration for :ref:`access logs <arch_overview_access_logs>`
-  // Output the access log for the administration server.
+  // emitted by the administration server. This is only required if
```

Comment:
Why is it required? Can't the access logs just be empty which would effectively end up being dev null? Sorry that I didn't raise this before. :)

/wait-any

Reply:
I also found it odd as well that `access_log_path` was required property and we set it to `/dev/null` in various samples. So I don't mind changing it but I wanted to maintain the existing behavior which says that if if `access_log_path` can not be unspecified if an admin address is specified.

Reply:
I honestly don't know why this was the way it was. IMO it's fine to relax this (and remove the proto constraint if necessary). I would just document the change. Thank you!

/wait

Reply:
fixed in e8f5139fbb0a73266785a994b32db4a3bf2d4449. Since this part of the deprecation and this is a new property I think we need extra documentation.